src/components/Table/index.tsx

```diff
@@ -241,13 +244,14 @@ export default function Table<D extends object>({
     actionGroup = null,
     columnDefinitions = [],
     defaultGroups = [],
+    defaultColumnFilter = DefaultColumnFilter,
     defaultPageIndex = 0,
     defaultPageSize = DEFAULT_PAGE_SIZE,
     disableGroupBy = true,
     disablePagination = false,
     disableSettings = false,
     disableSortBy = false,
-    disableFilters = false,
+    disableFilters = true,
```

Comment:
Just notice this flab is also using to determine whether to show the search filter on the top. Now the search filter disappear for the default cases:

<img width="1180" alt="Screen Shot 2021-08-22 at 2 43 00 PM" src="https://user-images.githubusercontent.com/5362048/130342640-eccc9a53-0d72-40fa-9506-d6582df43039.png">

Let us use another flag (may be disableColumnFilters) to determine whether to show or not column filters.